policies/secure-development-lifecycle.md

```diff
@@ -0,0 +1,45 @@
+# Secure Development Lifecycle Policy
+
+## Purpose
+The secure development lifecycle describes the approach that is used to define, develop, deliver and maintain software. 
+
+## Scope
+This lifecycle is intended to be used by software teams including product managers, designers, developers and quality assurance to develop secure software products. This policy describes the lifecycle of feature development.
+
+## Policy
+This lifecycle integrates security into all aspects of application software development which is comprised of six phases. 
+
+1. Requirements Gathering and Planning:
```

Comment:
Shall we specify who is responsible for each phase?

Reply:
For the purpose of preparing for the ISO audit, I don't think that level is necessary. We could add it in future iterations.